vinyldns/resource_zone_test.go

```diff
@@ -125,18 +229,58 @@ func testAccVinylDNSZoneImportStateCheck(s []*terraform.InstanceState) error {
 
 	rs := s[0]
 
-	expName := "system-test."
+	expName := zName
+	name := rs.Attributes["name"]
+	if name != zName {
+		return fmt.Errorf("expected name attribute to be %s, received %s", expName, name)
+	}
+
+	expEmail := zEmailUpdated
+	email := rs.Attributes["email"]
+	if email != expEmail {
+		return fmt.Errorf("expected email attribute to be %s, received %s", expEmail, email)
+	}
+
+	if rs.Attributes["admin_group_id"] == "" {
+		return fmt.Errorf("expected admin_group_id attribute to have value")
+	}
+
+	return nil
+}
+
+func testAccVinylDNSZoneWithACLImportStateCheck(s []*terraform.InstanceState) error {
+	if len(s) != 1 {
+		return fmt.Errorf("expected 1 state: %#v", s)
+	}
+
+	rs := s[0]
+
+	expName := zName
 	name := rs.Attributes["name"]
-	if name != expName {
+	if name != zName {
 		return fmt.Errorf("expected name attribute to be %s, received %s", expName, name)
 	}
 
-	expEmail := "updated_email@foo.com"
+	expEmail := zEmailUpdated
 	email := rs.Attributes["email"]
 	if email != expEmail {
 		return fmt.Errorf("expected email attribute to be %s, received %s", expEmail, email)
 	}
 
+	if rs.Attributes["admin_group_id"] == "" {
+		return fmt.Errorf("expected admin_group_id attribute to have value")
+	}
+
+	accessLev := rs.Attributes[fmt.Sprintf("acl_rule.%s.access_level", zACLRuleUpdatedHash)]
+	if accessLev != "Delete" {
+		return fmt.Errorf("expected acl_rule access_level attribute to have value; got %s from state: %s", accessLev, rs.Attributes)
```

Comment:
this error message is a little off yea?  "expected acl_rule access_level attribute to have value Delete; got ...."

It indicates that it doesn't have a value, but the `if` check is whether it is `Delete`